docs/install/airgapped-clusters.md

```diff
@@ -1,259 +1,299 @@
 
-There are the following sets of images to pull that will be shown along the
-following steps.
+Below is a quick summary of the procedure that will be covered in this guide;
 
-- the Ondat operator image
-- the images in the StorageOSCluster definition that define the images for each
-  component of the cluster
-- (if applicable) the images to run Etcd as Pods using the Etcd operator
-  deployed by Ondat
+1. Install the Ondat kubectl plugin.
+1. Generate the Ondat deployment manifests for your use case.
+1. Push Ondat container images to your private registry.
+1. Modify the Ondat deployment manifests.
+1. Install Ondat onto your air-gapped cluster.
 
-## Step 1. Install storageos kubectl plugin
+## Prerequisites
 
-```bash
-curl -sSLo kubectl-storageos.tar.gz \
-    https://github.com/storageos/kubectl-storageos/releases/download/v1.1.0/kubectl-storageos_1.1.0_linux_amd64.tar.gz \
-    && tar -xf kubectl-storageos.tar.gz \
-    && chmod +x kubectl-storageos \
-    && sudo mv kubectl-storageos /usr/local/bin/ \
-    && rm kubectl-storageos.tar.gz
-```
+> ⚠️ Make sure you have met the minimum resource requirements for Ondat to successfully run. Review the main [Ondat prerequisites](/docs/prerequisites/) page for more information.
 
-> 💡 You can find binaries for different architectures and systems in [kubectl
-> plugin](https://github.com/storageos/kubectl-storageos/releases).
+> ⚠️ Make sure the following CLI utility is installed on your local machine and is available in your `$PATH`:
 
-## Step 2. Generate yaml manifests
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 
-### Option 1: With Etcd in Kubernetes
+> ⚠️ Make sure to add an [Ondat licence](/docs/operations/licensing/) after installing.
 
-The etcd cluster in Kubernetes requires a StorageClass. If you are running on a
-cloud provider, you can use existing StorageClasses for it, for example `gp3`
-(AWS) or `standard` (GCE). Or you can create the `local-path` StorageClass.
-It's recommended to give Etcd a backend disk that can sustain at least 800
-IOPS. It's best to use provisioned IOPS when possible, otherwise make sure the
-size of the disk is big enough to fulfil the IOPS requirement when performance
-depends on IOPS per GB.  For example, AWS would require a burstable EBS bigger
-than 256G to fulfil the IOPS requirement.
+> ⚠️ Make sure you have a running Kubernetes cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
 
-1. Create StorageClass (if you don't have one available)
+> ⚠️ Make sure your Kubernetes cluster uses a Linux distribution that is officially supported by Ondat as your node operating system and has the required LinuxIO related kernel modules are available for Ondat to run successfully.
```

Comment:
I would suggest to split the following out:
> Make sure your cluster has the required LinuxIO related kernel modules

Reply:
sure, I have made the split to;

> ⚠️ Make sure your Kubernetes cluster uses a Linux distribution that is officially supported by Ondat as your node operating system.
> 
> ⚠️ Make sure that the node operating system have the required LinuxIO related kernel modules are available for Ondat to run successfully.

Let me know if this is fine.

Reply:
LGTM